### PR TITLE
Add daris.html file to repo

### PR DESCRIPTION
**Pull request by TheCandiko**
I'm doing some practice to get use with GIT.

This is first commit using VIM editor on command line. This will be body.

Glorious Geckos are amazing!

### CODE REVIEW

**Comment by TheCandiko** [comment]
This is how we test!

File: daris.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Chingu Voyage-4</title>
```
Comment (TheCandiko) [inline daris.html]: Where is the group name? Please add group name to title.

Comment (TheCandiko) [reply daris.html]: We can reply.

**Comment by TheCandiko** [comment]
To reply general comment, make another comment.

**Review by TheCandiko — COMMENTED** [review]
OMG! I can't request changes from myself!
Change that title anyway, Daris.

File: daris.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+</head>
+<body>
+<div>
+  Hello Glorious Geckos!
```
Comment (TheCandiko) [inline daris.html]: If I want to put this comment into review...

File: daris.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+</div>
+</body>
+</html>
```
Comment (TheCandiko) [inline daris.html:11]: Ohh, first comment is pending. Second comment on review?

File: daris.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (TheCandiko) [inline daris.html:1]: I can see comments number on top-right corner, green Review changes button?

Comment (TheCandiko) [reply daris.html:1]: Can't see comment but can complete review with that green comment.

Comment (TheCandiko) [reply daris.html:1]: Changes has been made.

**Comment by TheCandiko** [comment]
Hello Glorious Geckos! @dragonraij @hiralmakwana30 @shalinidc 
We can mention our names and summon each other.
This is my PR while I was trying commands and functionalities I spoke with myself. Take look, I believe there is a link in some of the comments?! Dunno, really. :confused:
By the way on GitHub while you are typing you can use:
- Markdown syntax: go [here](https://guides.github.com/features/mastering-markdown/) to learn it!
- Emojis :bowtie:
  - Go for [**emoji-cheat-sheet**](https://www.webpagefx.com/tools/emoji-cheat-sheet/)
- There was another thing, but... :astonished:
- Make me a review. I've done my homework, already!!! I need approval!!!

**Review by shalinidc — APPROVED** [review]

**Comment by shalinidc** [comment]
Woah !! 👍

**Pull request merged**